src/avatar/loading-dots/motion.scss

```diff
@@ -0,0 +1,87 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use '../../internal/styles/' as styles;
+@use '../../internal/styles/tokens' as awsui;
+
+.gen-ai {
+  &::after {
+    content: '';
+    position: absolute;
+    inline-size: inherit;
+    block-size: inherit;
+    inset-block-start: 50%;
+    inset-inline-start: 50%;
+    transform: translate(-50%, -50%);
+    background: awsui.$color-background-avatar-gen-ai;
+    @include styles.with-motion {
+      animation: gradientMove 3.6s infinite cubic-bezier(0.7, 0, 0.3, 1);
```

Comment:
We normally take motion durations from tokens like motion-duration-complex, should we introduce a new one here?

Reply:
I think one of the reasons is to support the reduced motion mode.